src/main/java/com/salesforce/dataloader/process/ProcessRunner.java

```diff
@@ -243,7 +253,11 @@ private static boolean validateCmdLineArgs (String[] args) {
     }
 
     private static void topLevelError(String message, Throwable err) {
-        logger.fatal(message, err);
+        if (logger == null) {
+            System.err.println(message);
```

Comment:
why System.err instead of System.out are there 2 different destinations? Looks like all other print statements go to System.our even for error conditions.

Reply:
I would switch others that are using System.out to print error message to System.err. However, that would be a different fix.

Reply:
updated to consistently use stderr when logging an error message prior to initializing Logger.